.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/devcontainers/python:3
+
+RUN python -m pip install --upgrade pip \
+    && python -m pip install pytest pytest-cov \
+    && python -m pip install 'flit>=3.8.0'
+
+ENV FLIT_ROOT_INSTALL=1
+
+COPY pyproject.toml README.rst ./
+RUN mkdir -p flit \
+    && python -m flit install --only-deps --deps develop \
+    && rm -r pyproject.toml README.rst flit
+
+// ENV FLIT_ALLOW_INVALID=1
+
+// COPY pyproject.toml .
+// RUN python -m flit install --only-deps --deps develop \
+//     && rm -r pyproject.toml
```

Comment:
```suggestion
    && python -m pip install 'flit>=3.8.1'

ENV FLIT_ROOT_INSTALL=1
ENV FLIT_ALLOW_INVALID=1

COPY pyproject.toml .
RUN python -m flit install --only-deps --deps develop \
    && rm -r pyproject.toml
```

Reply:
after this PR is completed, we can update the docker file.
https://github.com/pypa/flit/pull/631